Src/h19.cpp

```diff
@@ -3,12 +3,36 @@
 /// \date Apr 12, 2009
 /// \author Mark Garlanger
 ///
+#include "config.h"
+#include <iostream>
+#include <signal.h>
+#include <stdlib.h>
+
+#if OGL
+#ifdef __APPLE__
+#include <OpenGL/gl.h>
+#include <OpenGL/glu.h>
+#include <GLUT/glut.h>
```

Comment:
I fixed this in master-next.

Fixed other problems you pointed out on that branch, too.


On 02/07/2016 12:21 AM, Mark Garlanger wrote:
>
> In Src/h19.cpp 
> <https://github.com/mgarlanger/VirtualH89/pull/4#discussion_r52113088>:
>
> > @@ -3,12 +3,36 @@
> >  /// \date Apr 12, 2009
> >  /// \author Mark Garlanger
> >  ///
> > +#include "config.h"
> > +#include <iostream>
> > +#include <signal.h>
> > +#include <stdlib.h>
> > +
> > +#if OGL
> > +#ifdef __APPLE__
> > +#include <OpenGL/gl.h>
> > +#include <OpenGL/glu.h>
> > +#include <GLUT/glut.h>
>
> duplicate includes, glut.h was already included.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mgarlanger/VirtualH89/pull/4/files#r52113088>.
>